plugins/dev-scripts/config/webpack.config.js

```diff
@@ -120,6 +120,7 @@ module.exports = (env) => {
       // Run the linter.
       !env.skipLint && new ESLintPlugin({
         cache: true,
+        cacheLocation: path.join('node_modules/.cache/eslint/'),
```

Comment:
Isn't the more standard way to do this to pass each directory as a different parameter? E.g:

```suggestion
        cacheLocation: path.join('node_modules', '.cache', 'eslint'),
```